src/nav/apm.yml

```diff
@@ -1,70 +1,13 @@
 title: APM agents
 path: /docs/apm
 pages:
-  - title: Getting started
+  - title: Get started with APM
```

Comment:
@bradleycamacho: Austin made a great point about focusing on what problem someone is trying to solve. There's a way in which the "APM" acronym doesn't serve us very well. It hides all the good words "application" "performance" "monitoring"! :D

Here are some alternatives (you may be able to swap "app" for "application" for brevity):
* Monitor your application's performance
* Start monitoring your application's performance
* Get started monitoring your application's performance
* Learn how to monitor your application
* Monitor your application

Thinking outside the box a little:
* Meet APM
* Choose your monitor
* Choose your application monitor
* Monitor and fine-tune your performance
* Start monitoring your app
* Why APM matters